src/main/webapp/app/shared/modal/SimpleConfirmModal.tsx

```diff
@@ -26,6 +27,7 @@ export const SimpleConfirmModal: React.FunctionComponent<{
         </Button>
         <Button
           variant="primary"
+          disabled={props.confirmDisabled}
```

Comment:
If `props.confirmDisabled` is not passed, then we should default to false here

Reply:
Reactstraps button takes an optional disabled prop as well, so we can just pass it down. If it is null for our component, it will also be null for the button component which is the correct behavior.

Reply:
As logn as @bprize15 you verified the behavior.